### ISSUE TITLE
Is the entire dataset used for pre-training?

### ISSUE THREAD

**Issue by ibarrien** [HobbitLong/SupContrast#82]
From [this line](https://github.com/HobbitLong/SupContrast/blob/8d0963a7dbb1cd28accb067f5144d61f18a77588/main_supcon.py#L158
) it looks like the entire cifar10 dataset might be used for pre-training.
Also, what does the "96.0%" quoted accuracy mean (dataset-wise) for cifar10 with supcon?

**Comment by HobbitLong** [comment]
By default, it will only return the `train` set, instead of `train + val`.

"96.0" is the accuracy on validation set.
